pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -5,11 +5,11 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "aws-sam-cli";
-  version = "1.53.0";
+  version = "1.78.0";
```

Comment:
I've pulled in the 4 commits from this branch into a local branch and running `sam --version` outputted `SAM CLI, version 1.53.0`.

Still investigating to see if that's a fault of this PR or not.

Reply:
I tested this PR locally, and correctly got version 1.78.0 as a version.
```
nix-shell -p '(import ./default.nix {}).aws-sam-cli'
❯ sam --version
SAM CLI, version 1.78.0
```